Snakefile

```diff
@@ -1089,7 +1089,7 @@ rule prepare_sector_network:
     input:
         network=RESDIR
         + "prenetworks/elec_s{simpl}_{clusters}_ec_l{ll}_{opts}_{sopts}_{planning_horizons}_{discountrate}_{demand}_presec.nc",
-        costs="resources/" + RDIR + "costs_{planning_horizons}.csv",
+        costs=COSTS,
```

Comment:
The side effect here is that in overnight, the costs is fixed to the cost file in the data or resource folder.
In myopic, it is no more going to change across time horizons unfortunately